test/browser/features/fixtures/packages/error-correlation/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "error-correlation",
+    "private": true,
+    "scripts": {
+        "build": "rollup --config ../rollup.config.mjs",
+        "clean": "rm -rf dist"
+    },
+    "dependencies": {
+        "@bugsnag/browser": "^7.25.0-alpha.0"
```

Comment:
is there a particular reason for using the `@bugsnag/browser` dependency directly here and not `@bugsnag/js`?

Reply:
only that it's in the `browser` fixture, and not node, meaning for a smaller install

Reply:
ah okay, makes sense 👍🏾